server/graphql/v2/input/UserCreateInput.js

```diff
@@ -0,0 +1,9 @@
+import { GraphQLInputObjectType, GraphQLNonNull, GraphQLString } from 'graphql';
+
+export const UserCreateInput = new GraphQLInputObjectType({
```

Comment:
@znarf from the new nomenclature, shouldn't that be `IndividualCreateInput`?

Reply:
Yeah, `IndividualCreateInput` is better.